### DISCUSSION TITLE
DataExtension methods missing?

### DISCUSSION THREAD

**Discussion by rscott78** [MarkPflug/Sylvan#235] [Q&A] [answered]
I'm probably missing something simple... but I can't seem to find the `AsDataReader()` method. In the code, I see that it's part of the Sylvan.Data.DataExtensions class, but that also doesn't appear to exist in the package that I installed.

For what it's worth, I'm installing this using NuGet into a .net 8 project. 

Thoughts?

**Comment by rscott78** [comment, marked as answer]
It turns out there's an additional package that I needed to install. It's Sylvan.Data. Would be handy to see that somewhere in the Readme, or perhaps make it part of the dependency list so that when someone goes searching for a CSV handler in Nuget, they'll get the appropriate packages automatically.

**Comment by MarkPflug** [comment]
This is mentioned in the documentation here:
https://github.com/MarkPflug/Sylvan/blob/d26baf5f312a57069a81abbc203053c863be02c8/docs/Csv/Examples.md?plain=1#L86-L88

If you feel it could be clarified or improved, a PR would be welcome.
